{
  "uid" : "4770adcd27a64faf",
  "name" : "test_broken",
  "fullName" : "test_allure_demo1#test_broken",
  "historyId" : "99a98c8a62e3433aed3093e5ad13987e",
  "time" : {
    "start" : 1594180877178,
    "stop" : 1594180877178,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "Exception: oops",
  "statusTrace" : "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure_demo1.py:19: Exception",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "test_allure_demo1"
  }, {
    "name" : "host",
    "value" : "zhi"
  }, {
    "name" : "thread",
    "value" : "5412-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_allure_demo1"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "4770adcd27a64faf.json",
  "parameterValues" : [ ]
}